README.md

```diff
@@ -7,15 +7,59 @@ Queensland Government has developed this plugin to be used with data.qld.gov.au
 
 #Features
 * Config values with SSM Parameter Store placeholders, ${ssm:/path/to/value}, will be replaced at runtime.
+* All SSM parameters under a prefix can be automatically converted into config entries.
 
 #Requirements
-* None
+* boto
+* boto3
 
 #Configuration
 ```
-ckan.plugins = ssm-config
+ckan.plugins = ssm_config
 ```
 
+IAM permissions similar to the following are needed:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "ssm:GetParameter",
+                "ssm:GetParameters",
+                "ssm:GetParametersByPath"
+            ],
+            "Resource": [
+                "arn:aws:ssm:*:*:parameter/config/CKAN/default/",
+                "arn:aws:ssm:*:*:parameter/config/CKAN/default/*",
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+Optional:
+
+```
+ckanext.ssm_config.region_name = <region>
+ckanext.ssm_config.prefix = /path/to/config/
```

Comment:
/path/to/config/ should be /config/CKAN/default/ to match the example iam rule you have